src/video/video.js

```diff
@@ -340,23 +340,34 @@
                 var parent = me.video.renderer.getScreenCanvas().parentNode;
                 var _max_width = Math.min(maxWidth, parent.width || window.innerWidth);
                 var _max_height = Math.min(maxHeight, parent.height || window.innerHeight);
-
-                if (settings.maintainAspectRatio) {
-                    // make sure we maintain the original aspect ratio
-                    var designRatio = me.video.renderer.getWidth() / me.video.renderer.getHeight();
-                    var screenRatio = _max_width / _max_height;
+                var designRatio = me.video.renderer.getWidth() / me.video.renderer.getHeight();
+                var screenRatio = _max_width / _max_height;
+                var sWidth = me.video.renderer.getHeight() * screenRatio;
+                var sHeight = me.video.renderer.getWidth() * (window.innerHeight / window.innerWidth);
```

Comment:
why use `window.innerHeight` and `window.innerWidth` here instead of the previous `_max_width` and `max_height` ? the former provides the max size of the window, the later provide the max size of the document element in which the canvas is, or if none the window size. 

note that I'm not sure my self, just asking to see if you are clearer on your side about it :P